src/components/Button/CloseButton.tsx

```diff
@@ -2,14 +2,12 @@ import React, { ButtonHTMLAttributes } from 'react'
 import CloseIcon from '../../icons/CloseIcon'
 import classNames from 'classnames'
 
-export interface CloseButtonProps
-  extends ButtonHTMLAttributes<HTMLButtonElement> {
+type CloseButtonProps = {
   textOnly?: boolean
-}
+} & Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'children'>
```

Comment:
FYI: I had strange behavior when Prettier was enabled in the VSCode. After reloading Prettier, it gave other strange behavior. But I believe that we no longer use Prettier

Reply:
I believe same change could be applied to `BackButton`

Reply:
Thank you for spotting that. I'll go change that component.